packages/components/flex/docs.mdx

```diff
@@ -0,0 +1,90 @@
+---
+name: Flex
+route: /components/flex
+menu: Components
+---
+
+import { Playground, Props } from "docz"
+import Flex from './index'
+import Box from "../box"
+
+# Box
+
+Box component
```

Comment:
Define what a box component does and how it can be used.

Reply:
will be added